blueprints/twiddle.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
As a separate issue, maybe we can have this automatically use the current ember-twiddle version?  I think this should be possible by using `git-repo-info` (or another related package) and add the `version` to the blueprint at runtime...